### DISCUSSION TITLE
How to set transmition power and SNR in 5G SA with ZMQ interfaces

### DISCUSSION THREAD

**Discussion by jjose001** [srsran/srsRAN_Project#931] [General Help]
I have set up a 5G SA network using open5GS with srsUE using ZMQ interfaces. How can I set the transmission power and therefore the SNR? The only related parameters I found are the Tx/RX gain, but changing these values does not change the SNR values as shown in grafana. I want to understand how these parameters work in srsRAN with ZMQ configuration. ¿Any help?

**Comment by pgawlowicz** [comment]
please check here https://github.com/srsran/srsRAN_Project/issues/757

**Reply by jjose001** [reply]
Thank you! 
I'm going to take it as a reference.
